particle.inf

```diff
@@ -38,6 +38,12 @@ CatalogFile=particle.cat
 ; Core
 %DEVICE_DESCRIPTION_CORE%=DriverInstall, USB\VID_1D50&PID_607D
 
+; Photon pre-0.6.0
+%DEVICE_DESCRIPTION_PHOTON_3%=DriverInstall, USB\VID_2B04&PID_C006&REV_0200
```

Comment:
I see my pre-0.6.0 Photon with this ID in the device manager. The 0.6.0 Photon has `USB\VID_2B04&PID_C006&REV_0250` but you use `USB\VID_2B04&PID_C006&MI_00`. Is that ok?